### PR TITLE
Player request to download resource pack

### PR DESCRIPTION
**Pull request by SciathGH**

### CODE REVIEW

File: Proxy/src/main/kotlin/net/horizonsend/ion/proxy/IonProxy.kt [hunk 1]
```diff
@@ -26,11 +26,7 @@ import net.horizonsend.ion.proxy.commands.discord.PlayerListCommand
 import net.horizonsend.ion.proxy.commands.discord.ResyncCommand
 import net.horizonsend.ion.proxy.commands.velocity.VelocityAccountCommand
 import net.horizonsend.ion.proxy.commands.velocity.VelocityInfoCommand
-import net.horizonsend.ion.proxy.listeners.velocity.DisconnectListener
-import net.horizonsend.ion.proxy.listeners.velocity.LoginListener
-import net.horizonsend.ion.proxy.listeners.velocity.PreLoginListener
-import net.horizonsend.ion.proxy.listeners.velocity.ProxyPingListener
-import net.horizonsend.ion.proxy.listeners.velocity.ServerConnectedListener
+import net.horizonsend.ion.proxy.listeners.velocity.*
```
Comment (Peter-Crawley) [inline Proxy/src/main/kotlin/net/horizonsend/ion/proxy/IonProxy.kt:29]: Wildcard import

**Review by Peter-Crawley — CHANGES_REQUESTED** [review]

**Comment by SciathGH** [comment]
I hate intellij

**Comment by Peter-Crawley** [comment]
> I hate intellij

Don't we all... at least its not Visual Studio.

**Comment by SciathGH** [comment]
Thats true

**Pull request merged**